### DISCUSSION TITLE
LDAP features how can i use in nfstream?

### DISCUSSION THREAD

**Discussion by imagineTh4t** [nfstream/nfstream#84] [Q&A]
I want to get SMB features(domain,username,hostname,etc.) and LDAP features from nDPI on nfstream. Can you help me about it ?

**Comment by aouinizied** [comment]
Currently nDPI structures contains the following fields.

* ndpi_flow->protos.kerberos.domain.
* ndpi_flow->protos.kerberos.hostname,
* ndpi_flow->protos.kerberos.username.

Do you want to expose this fields in NFStream?

Zied

**Reply by imagineTh4t** [reply]
Bro i need a LDAP features ? Do u know about extract LDAP features?
